app/submitter/submitter.py

```diff
@@ -94,8 +94,14 @@ def send_message(self, message, queue):
         try:
             self._connect()
             channel = self.connection.channel()
-            channel.queue_declare(queue=queue)
-            published = channel.basic_publish(exchange='', routing_key=queue, body=message_as_string, mandatory=True)
+            channel.queue_declare(queue=queue, durable=True)
+            published = channel.basic_publish(exchange='',
+                                              routing_key=queue,
+                                              body=message_as_string,
+                                              mandatory=True,
+                                              properties=pika.BasicProperties(
+                                                  delivery_mode=2
+                                              ))
```

Comment:
Does this also need to make use of publisher confirms? https://www.rabbitmq.com/confirms.html That will (according to  documentation) block until the message is persisted to disk. That would at least give us an option of redisplaying to the user to try submission again (not that we currently support this).

Otherwise LGTM 👍